shared-models/src/Publication/epub/EpubLayout.ts

```diff
@@ -1,10 +1,10 @@
-/* Copyright 2020 Readium Foundation. All rights reserved.
+/* Copyright 2021 Readium Foundation. All rights reserved.
  * Use of this source code is governed by a BSD-style license,
  * available in the LICENSE file present in the Github repository of the project.
  */
 
 /** Hint about the nature of the layout for the linked resources. */
-export enum EPUBLayout {
-  fixed = "fixed",
-  reflowable = "reflowable"
-}
\ No newline at end of file
+export enum EpubLayout {
```

Comment:
Isn't the convention to keep accronyms as uppercase?

```suggestion
export enum EPUBLayout {
```

Reply:
renamed